Mupa1/StackOverFlowLite1
# Ft landing page 162638962

__What does this PR do__
Adds landing page for the user

__Description of task to be completed__
Create a user-friendly landing page with the following features:-
1.  Brief description of site
2.  Links to relevant pages
3.  View of top questions

__How should this be manually tested__
1. Clone this repository
2. Cd into the StackOverFlowLie1 folder
3. Open index.html in your browser

__What are the relevant pivotal tracker stories__
#162638962 A User should be able to see the landing page

__Screenshot__
![landing_page](https://user-images.githubusercontent.com/44979953/50024794-931c2b00-fff4-11e8-954a-89d161dce5e0.png)
